platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblems.kt

```diff
@@ -163,6 +166,15 @@ class ConfigurationCacheProblems(
         }
     }
 
+    override fun forConfigurationCacheIncompatibleTask(trace: PropertyTrace): ProblemsListener {
+        explicitlyIncompatibleTasks.add(trace)
+        return object : AbstractProblemsListener() {
+            override fun onProblem(problem: PropertyProblem) {}
```

Comment:
🤔 We need to decide how this behavior will be different from Task.notCompatibleWithConfigurationCache(). 
We should make tasks that disabling CC visible in CC report(and problems report?), I think